### ISSUE TITLE
Flood Infestors should need a small timer to infect, or a lower health threshold.

### ISSUE THREAD

**Issue by Nightspark43** [HaloSpaceStation/HaloSpaceStation13#658]
It seems a little, cheap, for flood infestors to be able to almost instantly infect a player unavoidably by just going near when at nearly full health, so, I'm suggesting one of two things, one, the health threshold should be 50% health, a much more significant chunk of health, OR, the infestor instead freezes the target with a, 2-5 second timer, visible to everyone, and killing the infestor frees the victim, but it deals a good bit of damage to the one freed cause you just shot a creature out your friend's skin, making it easier for another infestor to come along and infect them.

Basically, make it a little harder for infestors to kill people instantly.

**Comment by X0-11** [comment]
Lorewise, infestors instantly burrowed through the person, ignoring armour. The insta-infect change is meant to make the flood something to avoid.

I actually tested a 50^% health threshold when first implementing the infection-ability, and a 50% health threshold was more-than-often enough to paincrit the person.

Infestors are meant to make you keep your distance instead of rushing in.

**Comment by Nightspark43** [comment]
That doesn't work with carriers, considering they throw infestors in all 8 directions almost the entire screen away.

**Comment by X0-11** [comment]
Carriers are present to make players not just throw ammo down-range towards the possible targets, so I'd say that's intended.

**Comment by Nightspark43** [comment]
Then what about the second option, it saves from the issue as well, the issue being, Flood are currently OP due to the infection being instant, because gameplay is hopefully a little higher on priority than simple lore accuracy.

**Comment by X0-11** [comment]
A threshold of 50% is too low, most times people succumb to the pain from the wounds at that stage.

**Comment by Nightspark43** [comment]
Second option
the infestor instead freezes the target with a, 2-5 second timer, visible to everyone, and killing the infestor frees the victim, but it deals a good bit of damage to the one freed cause you just shot a creature out your friend's skin, making it easier for another infestor to come along and infect them.

**Comment by X0-11** [comment]
Being able to remove the infestor reduces their effectiveness as an area-deterrent. Are carriers the main issue when fighting infection forms?

**Comment by Nightspark43** [comment]
Mostly, yeah, just being able to see them puts you at risk unless you can fight them from behind cover, which isn't an option when admins spawn them.

**Comment by BDpuffy420** [comment]
non-issuse
